heureka/ui/src/App.js

```diff
@@ -5,7 +5,11 @@
 
 import React, { useEffect } from "react"
 import styles from "./styles.scss"
-import { AppShell, AppShellProvider, CodeBlock } from "juno-ui-components"
+import {
+  AppShell,
+  AppShellProvider,
+  CodeBlock,
+} from "@cloudoperators/juno-ui-components"
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query"
 import { MessagesProvider } from "messages-provider"
```

Comment:
Please also update imports for:
- communicator -> @cloudoperators/juno-communicator
- messages-provider -> @cloudoperators/juno-messages-provider
- url-state-provider -> @cloudoperators/juno-url-state-provider